### PR TITLE
docs: Fix type definition code broken by v0.24

### LINKED ISSUE #154: docs: Fix type definition code broken by v0.24 in multiple-functions.md
The code example in multiple-functions.md mentioning declaring types on lines 33 and 231 reads

```rust
pub type Frame<'a> = ratatui::Frame<'a, CrosstermBackend<std::io::Stderr>>;
```

[As of v0.24](https://ratatui.rs/highlights/v0.24.html#frame-no-more-generics-) these lines should read

```rust
pub type Frame<'a> = ratatui::Frame<'a>;
```

Discussed first here: https://github.com/ratatui-org/ratatui-book/discussions/151#discussioncomment-7524920

### PR DESCRIPTION
**Pull request by markalanboyd**
- Fixes #154

### CODE REVIEW

**Comment by markalanboyd** [comment]
Not sure why checks failed - did I cause that?

File: src/tutorial/counter-app/multiple-functions.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ By defining custom types and aliases, we can simplify our code and make it more
 ```rust
 type Err = Box<dyn std::error::Error>;
 type Result<T> = std::result::Result<T, Err>;
-pub type Frame<'a> = ratatui::Frame<'a, CrosstermBackend<std::io::Stderr>>;
+pub type Frame<'a> = ratatui::Frame<'a>;
```
Comment (kdheepak) [inline src/tutorial/counter-app/multiple-functions.md]: I believe we can drop this type definition entirely (notice the LHS and RHS are the same).

We'll have to update the imports in an earlier block to bring `Frame` into scope, i.e. `use ratatui::prelude::Frame`.

**Comment by kdheepak** [comment]
Thanks for making a PR! Don't worry about the check that fails, that's something we need to fix.

File: src/tutorial/counter-app/multiple-functions.md (hunk 1 above)
Comment (kdheepak) [inline src/tutorial/counter-app/multiple-functions.md]:
```suggestion
```

File: src/tutorial/counter-app/multiple-functions.md [hunk 2]
```diff
@@ -228,7 +227,7 @@ use ratatui::{
   widgets::Paragraph,
 };
 
-pub type Frame<'a> = ratatui::Frame<'a, CrosstermBackend<std::io::Stderr>>;
+pub type Frame<'a> = ratatui::Frame<'a>;
```
Comment (kdheepak) [inline src/tutorial/counter-app/multiple-functions.md]:
```suggestion
```

**Pull request merged**